### PR TITLE
refactor(ingress-egress-tracker): remove unnecessary fields

### PR DESCRIPTION
**Pull request by acdibble**
# Pull Request

Closes: PRO-xxx

## Checklist

Please conduct a thorough self-review before opening the PR.

- [ ] I am confident that the code works.
- [ ] I have updated documentation where appropriate.

## Summary

*Please include a succinct description of the purpose and content of the PR. What problem does it solve, and how? Link issues, discussions, other PRs, and anything else that will help the reviewer.*

### CODE REVIEW

File: api/bin/chainflip-ingress-egress-tracker/src/main.rs [hunk 1]
```diff
@@ -22,24 +21,13 @@ async fn start(
...
 
 	let client = redis::Client::open(settings.redis_url.clone()).unwrap();
-	let mut store = RedisStore::new(client.get_multiplexed_tokio_connection().await?);
+	let store = RedisStore::new(client.get_multiplexed_tokio_connection().await?);
 
-	// Broadcast channel will drop old messages when the buffer is full to
-	// avoid "memory leaks" due to slow receivers.
-	const EVENT_BUFFER_SIZE: usize = 1024;
-	let (witness_sender, _) =
-		tokio::sync::broadcast::channel::<state_chain_runtime::RuntimeCall>(EVENT_BUFFER_SIZE);
```
Comment (acdibble) [inline api/bin/chainflip-ingress-egress-tracker/src/main.rs:31]: no longer need to multicast the witnessing calls to different websockets so i removed the channel logic

File: api/bin/chainflip-ingress-egress-tracker/src/witnessing/btc_mempool.rs [hunk 2]
```diff
@@ -15,19 +15,30 @@ use std::{
...
 }
 
+impl Serialize for QueryResult {
+	fn serialize<S: serde::Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
+		use serde::ser::SerializeMap;
+
+		let mut map = serializer.serialize_map(Some(3))?;
+		map.serialize_entry("confirmations", &self.confirmations)?;
+		map.serialize_entry("tx_hash", &self.tx_hash)?;
+		map.serialize_entry("value", &format!("0x{:x}", self.value))?;
```
Comment (acdibble) [inline api/bin/chainflip-ingress-egress-tracker/src/witnessing/btc_mempool.rs:33]: hex encode the u64

File: api/bin/chainflip-ingress-egress-tracker/src/witnessing/state_chain.rs [hunk 3]
```diff
@@ -15,46 +15,22 @@ use serde::Serialize;
 use utilities::rpc::NumberOrHex;
 
 #[derive(Serialize)]
-pub struct WitnessAsset {
-	pub chain: ForeignChain,
```
Comment (acdibble) [inline api/bin/chainflip-ingress-egress-tracker/src/witnessing/state_chain.rs:19]: the chain is part of the key in redis so it doesn't need to be serialized

File: api/bin/chainflip-ingress-egress-tracker/src/witnessing/state_chain.rs [hunk 4]
```diff
@@ -15,46 +15,22 @@ use serde::Serialize;
...
-
-impl From<cf_chains::assets::btc::Asset> for WitnessAsset {
-	fn from(asset: cf_chains::assets::btc::Asset) -> Self {
-		Self { chain: ForeignChain::Bitcoin, asset: asset.into() }
-	}
-}
-
-#[derive(Serialize)]
-#[serde(tag = "deposit_chain")]
+#[serde(untagged)]
```
Comment (acdibble) [inline api/bin/chainflip-ingress-egress-tracker/src/witnessing/state_chain.rs:18]: this tag is part of the key in redis so it doesn't need to be serialized

**Review by niklasnatter — APPROVED** [review]

**Pull request merged**